### PR TITLE
fixes xeno headslugging not working

### LINKED ISSUE #23316: Headslugs if they egg a xenomorph runtime to shit
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

doesnt matter

### Issue Description

The egg runtimes as it tries to get dna from the xenomorph, and later runtimes when bursting as it can not fracture the chest

### What did you expect to happen?

Them to burst from the xenomorph without runtime

### What happened instead?

runtimes. Could have sworn it made a dna runtime too but could not find it

### Why is this bad/What are the consequences?

Should have a failsafe to prevent this issue and just give them their original dna rather than trying to absorb the xeno dna / fracture runtime

### Steps to reproduce the issue.

Headslug a xenomorph
Wait.
Runtime.

### When did the problem start happening?

Unknown prob forever

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
fixes #23316 

## Why It's Good For The Game
Intended behavior is good, runtimes are bad

## Testing
Egged a skrell and an xeno

## Changelog
:cl:
fix: xenos can be properly headslug'd again
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/organ.dm [hunk 1]
```diff
@@ -50,21 +50,18 @@
 /obj/item/organ/proc/update_health()
 	return
 
-/obj/item/organ/New(mob/living/carbon/holder, datum/species/species_override = null)
+/obj/item/organ/New(mob/living/carbon/human/holder, datum/species/species_override = null)
```
Comment (lewcc) [inline code/modules/surgery/organs/organ.dm:53]: Are you sure about this? Xenomorphs do have organs

Comment (GDNgit) [reply code/modules/surgery/organs/organ.dm:53]: Xenos have organs but they don't have DNA

**Review by DGamerL — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/headslug.dm [hunk 2]
```diff
@@ -108,11 +108,14 @@
...
-	chest.fracture()
-	chest.disembowel()
+	if(ishuman(owner))
+		owner.apply_damage(300, BRUTE, BODY_ZONE_CHEST)
+		owner.bleed(BLOOD_VOLUME_NORMAL)
+		var/obj/item/organ/external/chest = owner.get_organ(BODY_ZONE_CHEST)
+		chest.fracture()
+		chest.disembowel()
+	else
+		owner.gib()
```
Comment (S34NW) [inline code/modules/mob/living/simple_animal/hostile/headslug.dm:118]:
```suggestion
	if(!ishuman(owner))
		owner.gib()
		return
		
	owner.apply_damage(300, BRUTE, BODY_ZONE_CHEST)
	owner.bleed(BLOOD_VOLUME_NORMAL)
	var/obj/item/organ/external/chest = owner.get_organ(BODY_ZONE_CHEST)
	chest.fracture()
	chest.disembowel()
		
```

**Pull request merged**